ihmc-humanoid-robotics/src/main/java/us/ihmc/humanoidRobotics/communication/ControllerFootstepQueueMonitor.java

```diff
@@ -120,6 +123,16 @@ public FramePose3DReadOnly getLastFootstepQueuedOnOppositeSide(RobotSide candida
       return previousFootstepPose;
    }
 
+   private void acceptWalkingControllerFailureStatusMessage(WalkingControllerFailureStatusMessage message)
+   {
+      robotFalling.set(true);
+   }
+
+   public boolean getRobotFalling()
```

Comment:
This is polling the field
```suggestion
   public boolean pollRobotFalling()
```